### ISSUE TITLE
NullReferenceException when running 'codegen write' Oakton command

### ISSUE THREAD

**Issue by luber** [JasperFx/marten#3135]
I have asp.net core application with an Autofac IoC container.
When I try run `codegen write` command from command line the next exception is logged after the generation if finished:
```
#24 135.5 [red]ERROR:[/]System.NullReferenceException: Object reference not set to an instance of an 
#24 135.8 object.
#24 135.8   at async ValueTask Marten.Events.EventGraph.DisposeAsync()                    
#24 135.8   at async ValueTask Autofac.Core.Disposer.DisposeAsync(bool disposing)         
#24 135.8   at async ValueTask Autofac.Core.Lifetime.LifetimeScope.DisposeAsync(bool      
#24 135.8      disposing)                                                                 
#24 135.8   at async ValueTask Autofac.Core.Container.DisposeAsync(bool disposing)        
#24 135.8   at async ValueTask                                                            
#24 135.8      Autofac.Extensions.DependencyInjection.AutofacServiceProvider.DisposeAsync(
#24 135.8      )                                                                          
#24 135.8   at async ValueTask Microsoft.Extensions.Hosting.Internal.Host.                
#24 135.8      <DisposeAsync>g__DisposeAsync|21_0(object o)                               
#24 135.8   at async ValueTask Microsoft.Extensions.Hosting.Internal.Host.DisposeAsync()  
#24 135.8   at void Microsoft.Extensions.Hosting.Internal.Host.Dispose()                  
#24 135.8   at bool JasperFx.CodeGeneration.Commands.GenerateCodeCommand.Execute(         
#24 135.8      GenerateCodeInput input)                                                   
#24 135.8   at Task<bool> Oakton.OaktonCommand`1.Oakton.IOaktonCommand.Execute(object     
#24 135.8      input)                                                                     
#24 135.8   at Task<bool> Oakton.CommandRun.Execute()                                     
#24 135.8   at async Task<int> Oakton.CommandExecutor.execute(CommandRun run)    
```

It looks like `ValueTask Marten.Events.EventGraph` is already disposed when the Autofac IoC container is trying to dispose the lifetime scope

.net version: 8.0
Referenced packages:
```
        <PackageReference Include="Autofac" Version="8.0.0" />
        <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="9.0.0" />

        <PackageReference Include="Marten" Version="6.4.1" />
        <PackageReference Include="Marten.CommandLine" Version="6.4.1" />
```

**Comment by Hawxy** [comment]
This got fixed yesterday in v7.7

**Comment by jeremydmiller** [comment]
Yeah, sorry, without very good cause and let's be honest, paid support, it's very unlikely we patch the v6 branch.
